src/qdarts/plotting.py

```diff
@@ -101,10 +101,10 @@ def get_CSD_data(
     simulation: AbstractCapacitiveDeviceSimulator,
     P: np.ndarray,
     v_0: np.ndarray,
-    lower_left: list[float],

```

Comment:
type union operator | is a python 3.10 extensension:
https://peps.python.org/pep-0604/